packages/ui/feature-builder-form-controls/src/lib/components/new-piece-properties-form/piece-properties-form.component.html

```diff
@@ -0,0 +1,275 @@
+<form [formGroup]="form" class="ap-flex ap-flex-col">
+  @for (
+    property of sortedPropertiesByRequired | objectToArray;
```

Comment:
This behavior we wanted to change since long time!


Is it feasible to just show them in the original order?


This blocker to implement a visual seperator in piece framework

Reply:
Check this https://github.com/activepieces/activepieces/pull/3206

Also lets do in another pr for sure